fre/pp/ppyaml_test/pp.yaml

```diff
@@ -21,10 +21,10 @@ rose-suite:
     do_analysis: False
 
   directories:
-    history_dir: "/archive/c2b/am5/2022.01/c96L65_am5f1a0r0_amip/gfdl.ncrc5-intel22-classic-prod-openmp/history"
-    pp_dir: "/archive/$USER/canopy/am5/c96L65_amip/gfdl.ncrc5-deploy-prod-openmp/pp"
+    history_dir: "/archive/c2b/$(stem)/2022.01/$(expname)/$(platform)-$(target)/history"
```

Comment:
We really want "/archive/c2b/$(stem)/$(name)/$(platform)-$(target)/history", with stem being "am5/2022.01" but we can't expand "stem" yet.